pkg/apps/blackduck/v2/deployer.go

```diff
@@ -129,7 +129,7 @@ func (hc *Creater) GetComponents(blackduck *blackduckapi.Blackduck) (*api.Compon
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		componentList.ReplicationControllers = append(componentList.ReplicationControllers, cfsslRc)
+		componentList.Deployments = append(componentList.Deployments, cfsslRc)
```

Comment:
Should we change cfsslRc to cfsslDep? In VsCode you can do this quickly by using the regular expression `Rc([,)])` and replacing it with `Dep$1`.

Reply:
@mphammer fixed.